hs/test-examples/nl-eval-errors/Err_Eval_IncompatibleStates.txt

```diff
@@ -1,3 +1,4 @@
 reachc: error: ./Err_Eval_IncompatibleStates.rsh:6:28:after expr stmt semicolon: incompatible states:
-  After First Message: False vs True
-  Participant Definitions: fromList [] vs fromList [("A",DLVar ./Err_Eval_IncompatibleStates.rsh:5:6:dot "A" Address 0)]
+Issues:
+  * Expected a publication to have been made by this point.
+  * Expected "A" to be a participant but there are none.
```

Comment:
I will be easy to modify this to be what we want, but I think I need to explain what these `pdvs` even are...

A participant appears in `pdvs` if it has been either `Participant.set()` or if it has `publish`ed. So, a good way to "explain" this is to say something like that, "Expected 'A' to have published a message or been set, but it hasn't."

Reply:
Ah okay, thanks. Is it correct to say that the participant has to have an associated address to be in `pdvs`? In this example, `A` has `publish`ed before `transfer`ing and it's not in `pdvs`

Reply:
Yes, being in `pdvs` /means/ the participant has an address (i.e. has acted publicly or been assigned)